Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM node:14
+WORKDIR /usr/src/app
+COPY package*.json ./
+RUN npm install
+COPY . .
+ENTRYPOINT ["npm", "run", "watch"]
```

Comment:
Maybe this should be combined with `CMD` so the container can be used for the three commands.